relay-server/src/actors/processor.rs

```diff
@@ -1747,16 +1759,15 @@ impl EnvelopeProcessorService {
         let scoping = state.envelope_context.scoping();
         let (enforcement, limits) = metric!(timer(RelayTimers::EventProcessingRateLimiting), {
             envelope_limiter
-                .enforce(&mut state.envelope, &scoping)
+                .enforce(&mut state.envelope, &scoping, &state.early_enforcement)
                 .map_err(ProcessingError::QuotasFailed)?
         });
 
         if limits.is_limited() {
             ProjectCache::from_registry()
-                .do_send(UpdateRateLimits::new(scoping.project_key, limits.clone()));
+                .do_send(UpdateRateLimits::new(scoping.project_key, limits));
         }
 
-        state.rate_limits = limits;
```

Comment:
Is this being removed because we already did not use it on master, or is the removal related to the PR?